config/config.go

```diff
@@ -110,6 +118,8 @@ type creationRule struct {
 	PGP               string
 	GCPKMS            string     `yaml:"gcp_kms"`
 	AzureKeyVault     string     `yaml:"azure_keyvault"`
+	VaultURI          string     `yaml:"hc_vault_transit_uri"`
+	Vault             []vaultKey `yaml:"hc_vault_transit"`
```

Comment:
As discussed in the parent PR, we will not be support two ways of doing this. We will only support URIs or structured data, but not both, and it must be the same through CLI flags and the config.

Reply:
Since all the other KMS are using URI, I believe going with URI would be the best choice for consistency.

Reply:
That sounds like the best option to me right now, yes.